apps/api/src/config/configuration.ts

```diff
@@ -38,6 +39,7 @@ export const configuration = (): Configuration => {
     elasticUrl: process.env.ELASTIC_URL,
     elasticUsername: process.env.ELASTIC_USERNAME,
     elasticPassword: process.env.ELASTIC_PASSWORD,
+    courseIndex: process.env.COURSE_INDEX_NAME,
```

Comment:
is this a required config (i.e. the app can't run without it)? If so, put it in requiredConfigs array below as well.